client/src/js/administration/components/SampleRights.js

```diff
@@ -26,30 +26,34 @@ export const SampleRights = props => {
             <BoxGroupSection>
                 <label className="control-label" style={{ width: "100%" }}>
                     <span>Sample Group</span>
-                    <Help pullRight>
-                        <p>
-                            <strong>None</strong>: samples are assigned no group and only
-                            <em> all users'</em> rights apply
-                        </p>
-                        <p>
-                            <strong>User's primary group</strong>: samples are automatically assigned the creating{" "}
-                            user's primary group
-                        </p>
-                        <p>
-                            <strong>Choose</strong>: samples are assigned by the user in the creation form
-                        </p>
-                    </Help>
                 </label>
 
-                <InputError
-                    type="select"
-                    value={props.sampleGroup}
-                    onChange={e => props.onChangeSampleGroup(e.target.value)}
+                <SelectBox
+                    onClick={() => props.onChangeSampleGroup("none")}
+                    active={props.sampleGroup === "none" ? true : ""}
                 >
-                    <option value="none">None</option>
-                    <option value="force_choice">Force choice</option>
-                    <option value="users_primary_group">User's primary group</option>
-                </InputError>
+                    <strong>None</strong>
+                    <p>
+                        samples are assigned no group and only
+                        <em> all users'</em> rights apply
+                    </p>
```

Comment:
- styling does is not consistent with other usages of `SelectBox`. Look at how you originally wrote `SelectBox` to fix this.
- distribute boxes horizontally